### PR TITLE
npd6: new aport

### PR DESCRIPTION
**Pull request by Ikke**

### CODE REVIEW

File: testing/npd6/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Contributor: Kevin Daudt <kdaudt@alpinelinux.org>
+# Maintainer: Kevin Daudt <kdaudt@alpinelinux.org>
+pkgname=npd6
+pkgver=1.1.0
+pkgrel=0
+pkgdesc="Neighbor proxy daemon for IPv6"
+url="https://www.ipsidixit.net/"
+arch="all"
+license="GPL-3.0+"
```
Comment (tanertas) [inline testing/npd6/APKBUILD]: Should be "GPL-3.0-or-later" according to https://spdx.org/licenses/

File: testing/npd6/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+makedepends="linux-headers"
+subpackages="$pkgname-doc $pkgname-openrc::noarch"
+options="!check"
```
Comment (tanertas) [inline testing/npd6/APKBUILD]: Need comment to reason why passing the tests. Like:
`options="!check" # No test suite`

File: testing/npd6/npd6.initd [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/sbin/openrc-run
+
+command="/usr/bin/npd6"
+command_args="--foreground"
+command_background=true
+pidfile="/var/run/npd6.pid"
```
[the remark this answers is not in the document]
Comment (tanertas) [reply testing/npd6/npd6.initd]: If this is a network depended service, requirements must be defined in initscrit.
Checkout:
https://github.com/alpinelinux/aports/blob/master/main/busybox-initscripts/dnsd.initd

File: testing/npd6/APKBUILD (hunk 1 above)
Comment (Ikke) [reply testing/npd6/APKBUILD]: I accidentally copied it from the deprecated section. Will fix it.

File: testing/npd6/npd6.initd (hunk 3 above)
[the remark this answers is not in the document]
Comment (Ikke) [reply testing/npd6/npd6.initd]: Thanks, will fix this.

File: testing/npd6/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+	make
+}
+
+package() {
+	cd "$builddir"
+	make install DESTDIR="$pkgdir"
+
+	rm -r "$pkgdir"/etc/init.d
+	install -Dm0755 "$srcdir"/npd6.initd "$pkgdir"/etc/init.d/npd6
+        install -Dm0644 "$srcdir"/npd6.confd "$pkgdir"/etc/conf.d/npd6
```
Comment (tanertas) [inline testing/npd6/APKBUILD]: There are trailing spaces instead of tab here.

File: testing/npd6/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+url="https://www.ipsidixit.net/"
+arch="all"
+license="GPL-3.0-or-later"
+makedepends="linux-headers"
+subpackages="$pkgname-doc $pkgname-openrc::noarch"
+options="!check" # No test suite present
+source="$pkgname-$pkgver.tar.gz::https://github.com/npd6/npd6/archive/$pkgver.tar.gz
+        remove-sysctl-h.patch
+        $pkgname.initd
+        $pkgname.confd"
```
Comment (tanertas) [inline testing/npd6/APKBUILD]: There are trailing spaces instead of tab above 3 lines.

**Comment by Ikke** [comment]
Whitespace issues have been fixed

**Pull request closed without merging**